internal-api/src/main/proto/com/makarytskyi/internalapi/commonmodels/order/order_cancellation_notification.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package com.makarytskyi.internalapi.commonmodels.order;
+
+import "com/makarytskyi/internalapi/commonmodels/order/order.proto";
+
+option java_multiple_files = true;
+
+message OrderCancellationUserNotification {
```

Comment:
Should we delete word `User`?

Does it matter having the word in the notification name?

Reply:
Yes, it is better to delete this word.